tests/urlpatterns/route_urls.py

```diff
@@ -0,0 +1,12 @@
+from django.conf.urls import include
+from django.urls import path, re_path
+
+from . import views
+
+urlpatterns = [
+    path('articles/2003/', views.empty_view, name='articles-2003'),
+    path('included_urls/', include('urlpatterns_reverse.included_urls')),
+    re_path(r'^regex/(?P<pk>[0-9]+)/$', views.empty_view, name='regex'),
+    path('', include('urlpatterns_reverse.extra_urls')),
```

Comment:
I think you misunderstood my comment. The idea is that the `urlpatterns_reverse` app shouldn't be used in the `urlpatterns` app.

Reply:
> I think you misunderstood my comment. 

Indeed :crying_cat_face: 

Should I also remove this one: https://github.com/django/django/blob/master/tests/urlpatterns/path_urls.py#L13 ?

Reply:
Oops, I didn't notice that. It could be done but as a separate commit since it's not related to this issue.

Reply:
https://github.com/django/django/pull/10657/commits/1aaff659437221b104e05fc94b91a4f060c8d8f9 should be that commit